doc/rst/developer/bestPractices/index.rst

```diff
@@ -0,0 +1,108 @@
+===============================
+Best Practices for Contributors
+===============================
+
+.. comment:
+
+   update the following toctree when the document is linked in here,
+   so that we can be sure everything is included.
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   GettingStarted
+   ContributorInfo
+   DCO
+   CompilerDebugging
+   CompilerIRTricks
+   ErrorWarningMessaging
+   RuntimeLibrary
+   GeneratedCode
+   GASNetOnDesktops
+   git
+   CHPL_DEVELOPER
+   TestSystem
+   SpellChecking
+   Potpourri
+   Valgrind
+   Sanitizers
+   Deprecation
+   NightlyTesting
```

Comment:
The file associated with this does not display "CHPL_NIGHTLY_CRON_RECIPIENT and CHPL_NIGHTLY_TEST_CONFIG_NAME" correctly in its second paragraph.  They need to be surrounded with two ` instead of just one